inst/extdata/mvp/istr.Rmd

```diff
@@ -0,0 +1,343 @@
+
+## Data 
+
+The tiltIndicator package comes with example datasets
+
+```{r}
+raw_istr_companies <- read.csv(here("data-raw", "istr_companies.csv"), sep = ";")
+
+
+raw_istr_companies |> 
+  glimpse()
+```
+
+```{r}
+istr_ep_weo <- readxl::read_xlsx((here('data/istr_ep_weo.xlsx')))
+glimpse(istr_ep_weo)
+```
+
+```{r}
+istr_weo_2022 <- readr::read_csv(here('data/istr_weo_2022.csv'), show_col_types = FALSE)
+glimpse(istr_weo_2022)
+```
+
+# Data Preperation
+```{r}
+reshaped_istr_companies <- istr_reshaping(raw_istr_companies)
+
+reshaped_istr_companies |> tibble::as_tibble()
+```
+
+# Analysis
+
+## Introduction
+
+The Input Product Sector Risk Indicator assesses the transition risk of the input products based on the sector’s emissions targets the input product belongs to. Those sector emission reduction targets vary across scenarios (e.g., net zero scenario or 1.5° scenario) and the time horizon (e.g., reduction needed in 2030, 2040, 2050 to achieve the targets). It, therefore, is similar to the Product Sector Risk Indicator but focuses on the input products that the company needs to produce its products.The input products are, for example, resources, packaging materials, energy and enabling services (such as tractor use on farm) to produce the product.
+
+If an input product belongs to a sector in which reduction targets are over 70%, it belongs to a high-risk sector and is classified as ‘high sector transition risk’. If the input product belongs to a sector in which reduction targets are between 30-70%, the product is classified into the category ‘medium sector transition risk’ and if it is below 30% it belongs to the category of ‘low sector transition risk’. If it does not belong to a sector that has decarbonization targets, it is classified as ‘no sector transition risk’.
+
+After assessing the input products for each product, they will be aggregated at company-level to derive what percentage of the input products required by the company to produce its products have high, medium and low sector transition risk. We, therefore, derive the company-level information.
+
+Please note that carbon emissions or emissions always mean CO2e.
+
+The goal of the transition risk MVP is to create a first draft of how the sector transition risk indicator for inputs would be build in code in order to convert it into code production easily in the future.
+
+The transition risk indicator consists of 5 main steps:
+
+- Step 1: Identifying the input products for each product
+- Step 2: Matching the input products to the sectors
+- Step 3: Identifying sector-specific emission reduction targets
+- Step 4: Categorising sector emission reduction targets into high, medium, and low sector transition risk
+- Step 5: Aggregating on the company-level
+
+## Step 1: Identifying 
+Use mapper that serves as a sector classification bridge between the Ecoinvent's' sectors and the WEO 2022 file.
+
+```{r}
+istr_companies <- istr_mapping(reshaped_istr_companies, istr_ep_weo) |> 
```

Comment:
TODO: Investigate  `istr_mapping()` to understand why the data becomes so large.